src/Components/App/App.js

```diff
@@ -18,7 +18,7 @@ class App extends Component {
       cards: [],
       selectedCard: {},
       isFavorite: false,
-      savedReadings: []
+      userSavedReadings: []
```

Comment:
Love the detail addition, makes it much easier to understand what data you are dealing with and why it is there!